schema/cassandra/temporal/schema.cql

```diff
@@ -94,6 +94,20 @@ CREATE TABLE tasks (
     'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
   };
 
+-- Stores task queue information such as user provided versioning data
+CREATE TABLE task_queue_user_data (
+  namespace_id    uuid,
+  task_queue_name text,
+  data            binary, -- temporal.server.api.persistence.v1.TaskQueueUserData as proto binary blob
+  version         int64,  -- Version of this row, used for optimistic concurrency
+  updated_at      timestamp,
+  -- task_queue_name is not a part of the parititioning key to allow iterating all task queues in a single namespace.
+  -- Access to this table should be infrequent enough that a single partition per namespace can be used.
```

Comment:
maybe comment that using only namespace as the partition imposes a limit on total task queue user data within one namespace (say in the few hundreds of MB)

Reply:
Will do.